script/texdoclib-cli.tlu

```diff
@@ -180,6 +166,15 @@ local function do_action(action)
         end
         texdoc.view.view_file(arg[1])
         os.exit(C.exit_ok)
+    elseif action and string.sub(action, 1, #'complete-') == 'complete-' then
+        local shell = string.gsub(action, 'complete%-', '')
+        if shell == 'zsh' then
+            texdoc.util.print_zsh_completion()
+            os.exit(C.exit_ok)
+        else
+            print(shell .. ' is not supported currently!')
```

Comment:
I think this should go to stderr rather than stdout. Please consider using `err_print()` just like:

https://github.com/TeX-Live/texdoc/blob/b8f7ebd9eb14ad57d9d3f04e21fecaad349f6cf5/script/texdoclib-cli.tlu#L177